src/Lumper.UI/ViewModels/VtfBrowser/VtfBrowserViewModel.cs

```diff
@@ -0,0 +1,125 @@
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Lumper.Lib.BSP.Struct;
+using Lumper.UI.Models;
+using Lumper.UI.Models.Matchers;
+using ReactiveUI;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace Lumper.UI.ViewModels.VtfBrowser;
+
+public partial class VtfBrowserViewModel : ViewModelBase
+{
+    public class Vtf : ReactiveObject
+    {
+        private Image<Rgba32>? _image;
+        private string _name = "";
+
+        public Image<Rgba32>? Image
+        {
+            get => _image;
+            set => this.RaiseAndSetIfChanged(ref _image, value);
+        }
+
+        public string Name
+        {
+            get => _name;
+            set => this.RaiseAndSetIfChanged(ref _name, value);
+        }
+    }
+
+    private double _dimensions = 128;
+
+    public double Dimensions
+    {
+        get => _dimensions;
+        set
+        {
+            this.RaiseAndSetIfChanged(ref _dimensions, value);
+            this.RaisePropertyChanged(nameof(MaxNameWidth));
+        }
+    }
+
+    public uint MaxNameWidth => (uint)_dimensions - 8;
```

Comment:
at least comment the magic number .. whats that for?

Reply:
well I thought the variable names would be enough but I guess without any context it's not that obvious, it's just to pad the vtf name by 8 pixels, tell me if it needs a comment

Reply:
its just a good idea in general to have comments or variables for magic numbers so people know what it is in the future